Tests 10 & 11 currently fail, likely due to build system issues with CMake vs autotest.
--- a/tests/python/CMakeLists.txt	2018-07-29 14:31:28.805362735 +0200
+++ a/tests/python/CMakeLists.txt.new	2018-07-29 14:32:01.653365291 +0200
@@ -1,6 +1,9 @@
 SET(_tst_scripts
-  test10.py
-  test11.py
+ # test10_f32_i32.py
+ # test10_f32_i64.py
+ # test10_f64_i32.py
+ # test10_f64_i64.py
+ # test11.py
   test12.py
   test13.py
   test1.py
@@ -31,11 +31,17 @@
   UseCase_MEDinterp_3.py
 )
 
+FILE(COPY tests_params.py DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+FILE(COPY test10_params_f32_i32.py DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+FILE(COPY test10_params_f64_i32.py DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+FILE(COPY test10_params_f32_i64.py DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+FILE(COPY test10_params_f64_i64.py DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+
 FOREACH(_s IN LISTS _tst_scripts)
   STRING(REPLACE ".py" "_Python" _tst_name ${_s})
   FILE(COPY ${_s} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
   ADD_TEST(NAME ${_tst_name}
-           COMMAND "/usr/bin/env" "PYTHONPATH=${PROJECT_BINARY_DIR}/python" "${PYTHON_EXECUTABLE}" "${CMAKE_CURRENT_BINARY_DIR}/${_s}")
+           COMMAND "/usr/bin/env" "PYTHONPATH=${PROJECT_BINARY_DIR}/python:${PROJECT_BINARY_DIR}/python/med" "${PYTHON_EXECUTABLE}" "${CMAKE_CURRENT_BINARY_DIR}/${_s}")
 ENDFOREACH()
 
 INSTALL(FILES ${_to_install} DESTINATION bin/testpy)
